### DISCUSSION TITLE
s3 backupstoragelocation access denied

### DISCUSSION THREAD

**Discussion by caogenfu** [vmware-tanzu/velero#4527] [Community support Q&A]
DESCRIPTION:
My aws access key works well when I use aws cli

ERROR LOG:
velero time="2022-01-13T06:33:32Z" level=error msg="Error listing backups in backup store" backupLocation=default controller=backup-sync error="rpc error: code = Unknown desc = AccessDenied: Access Denied status code: 403, request id: PWBFB4ZD21SR5MF3, host id: 7mMbrMlirZK3BFfm96GyQ74Swtq92URpnrsoLrkjZ7v917tT/HSVbDyIQOgM/V9I3RGTtvOgcyI=" error.file="/go/src/velero-plugin-for-aws/velero-plugin-for-aws/object_store.go:361" error.function="main.(*ObjectStore).ListCommonPrefixes" logSource="pkg/controller/backup_sync_controller.go:182"

**Comment by caogenfu** [comment]
> 
aws said that I have used an other iam role which I don't configured to send requeset.
Does velero use default credentials in kubernetes?

**Reply by qiuming-best** [reply]
403 error which means it has no permission to execute the AWS s3 ListObjectsV2Pages  function, (ListCommonPrefixes call s3 ListObjectsV2Pages func). Maybe your can check your current AWS credential permission @caogenfu
